server/src/main/java/org/elasticsearch/index/IndexSettings.java

```diff
@@ -193,6 +176,24 @@
     public static final Setting<Long> INDEX_SOFT_DELETES_RETENTION_OPERATIONS_SETTING =
         Setting.longSetting("index.soft_deletes.retention.operations", 0, 0, Property.IndexScope, Property.Dynamic);
 
+    /**
+     * Controls how long translog files that are no longer needed for persistence reasons
+     * will be kept around before being deleted. A longer retention policy is useful to increase
+     * the chance of ops based recoveries.
```

Comment:
Why don't we keep the note related to soft-deletes?

Reply:
I thought that the setting was just moved and didn't pay close enough attention to the comment :( 

Updated it now.